src/Model/CampaignDraft.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Mailjet\MailjetBundle\Model;
+
+/**
+ * https://dev.mailjet.com/email-api/v3/campaigndraft/
+ */
+class CampaignDraft {
+
+    const LOCALE_KEY = 'Locale';
+    const SENDER_KEY = 'Sender';
+    const SENDEREMAIL_KEY = 'SenderEmail';
+    const SUBJECT_KEY = 'Subject';
+    const CONTACTLISTID_KEY = 'ContactsListID';
+
+    //Mandatory properties (must be set through constructor)
+    protected $locale;
+    protected $sender;
+    protected $senderEmail;
+    protected $subject;
+    protected $contactsListID;
+    //Optional properties (array)
+    protected $optionalProperties = null;
+    //content
+    protected $content = null;
+    //id
+    protected $id = null;
+
+    /**
+     * @param $Locale
+     * @param $Sender
+     * @param $SenderEmail
+     * @param $Subject
+     * @param $ContactsListID
+     * @param optionalProperties
+     */
+    public function __construct($locale, $sender, $senderEmail, $subject, $contactsListID, $optionalProperties = null) {
```

Comment:
In case of non backward compatibility change I would agree, but this should never happen in Mailjet API. We'll only add new properties, never remove any, or we'll version.

I don't believe it's the bundle responsibility to manage mandatory/optional properties. Plus, it's half implemented since there's actually no check done on that.